workers/unity/Packages/com.improbable.gdk.standardlibraryutils/CoordinatesExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Improbable;
+using UnityEngine;
+
+public static class CoordinatesExtensions
```

Comment:
Is it meant to be not in a namespace? (I like not having to require a `using` but just to confirm)

Reply:
Oh woops. That was an oversight 😬

Reply:
I vote for "feature, not a bug" :P Or... to perhaps put in the same namespace as Coordinates as a nice trick?

Reply:
I mean I can leave it namespace-less if people want

Reply:
Or just put it in `Improbable` as that will be imported whenever you use `Coordinates, Vector3f, or Vector3d` I think

Reply:
put it in a `Improbable` namespace